app/models/miq_server/log_management.rb

```diff
@@ -82,6 +84,17 @@ def last_log_sync_message
     last_log.try(:miq_task).try!(:message)
   end
 
+  def include_automate_models_and_dialogs?(value)
+    case value
+    when true
+      true
+    when false
+      false
+    when nil
+      Settings.log.collection.include_automate_models_and_dialogs
+    end
```

Comment:
haha, I was debating using that guard clause because it wasn't really a guard clause.  I'm indifferent to which way.  

flog goes from 6.5 to 5.1 so I'm good with changing it.